qiskit_experiments/library/hamiltonian/heat_base.py

```diff
@@ -0,0 +1,156 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Base Class for general Hamiltonian Error Amplifying Tomography experiments.
+"""
+
+from typing import List, Tuple, Optional
+
+from qiskit import circuit, QuantumCircuit
+from qiskit.providers import Backend
+
+from qiskit_experiments.framework import BaseExperiment, Options
+from .heat_analysis import HeatElementAnalysis
+
+
+class HeatElement(BaseExperiment):
+    """Base class of HEAT experiment elements.
+
+    # section: overview
+
+        Hamiltonian error amplifying tomography (HEAT) is designed to amplify
+        the dynamics of entangler circuit on a qubit along a specific axis.
+
+        The basic form of HEAT circuit is represented as follows.
+
+        .. parsed-literal::
+
+                                    (xN)
+                 ┌───────┐ ░ ┌───────┐┌───────┐ ░ ┌───────┐
+            q_0: ┤0      ├─░─┤0      ├┤0      ├─░─┤0      ├───
+                 │  prep │ ░ │  heat ││  echo │ ░ │  meas │┌─┐
+            q_1: ┤1      ├─░─┤1      ├┤1      ├─░─┤1      ├┤M├
+                 └───────┘ ░ └───────┘└───────┘ ░ └───────┘└╥┘
+            c: 1/═══════════════════════════════════════════╩═
+                                                            0
+
+        The circuit in the middle is repeated ``N`` times to amplify the Hamiltonian
+        coefficients along a specific axis of the second qubit. The ``prep`` circuit is
+        carefully chosen based on the generator of the ``heat`` gate under consideration.
+        The ``echo`` and ``meas`` circuits depend on the axis of the error to amplify.
+        Only the second qubit is measured following to the projection in the ``meas`` circuit.
```

Comment:
```suggestion
        Only the second qubit is measured.
```
The end of the sentence was unclear and perhaps not needed.